### DISCUSSION TITLE
Support Firefox extension

### DISCUSSION THREAD

**Discussion by reliek21** [dailydotdev/daily#1659] [Feature Requests]
Will it be possible to add support for Firefox extensions? It's crazy sometimes only having Google installed to keep up with the news I follow. I love the product, but sometimes I try to look for alternatives that have a Firefox extension or even a desktop version. It would be great if you provided support for Firefox extensions.

**Comment by idoshamun** [comment]
Hi @reliek21,

Please have a look here: https://github.com/dailydotdev/daily/issues/1320
